assets/tests/abac_tests.yaml

```diff
@@ -786,4 +786,43 @@ tests:
               user: user:jon
             context:
               "x": -1.79769313486231570814527423731704356798070e+309
-            errorCode: 2000
\ No newline at end of file
+            errorCode: 2000
+  - name: exclusion_resolves_deterministically
```

Comment:
This should not be an integration test. It should be a unit test that runs a reasonable number of times to guarantee that we were not just lucky if it passed

Reply:
Added a unit test case to the `TestResolveCheckDeterministic` test  in the `internal/graph/check_test.go` file